src/index.html

```diff
@@ -2,77 +2,101 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Product</title>
-
-    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
     <link
       rel="preconnect"
       href="https://fonts.gstatic.com"
       crossorigin="use-credentials"
     >
-    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
-
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap"
+      rel="stylesheet"
+    >
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
-        <img src="./images/logo.png" alt="Moyo logo">
+      <a
+        class="header__logo"
+        href="#"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+        >
       </a>
-
       <nav class="nav">
         <ul class="nav__list">
           <li class="nav__item nav__item--is-active">
-            <a href="" class="nav__link">Apple</a>
+            <a
+              href="#"
+              class="nav__link"
+            >Apple</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Samsung</a>
+            <a
+              href="#"
+              class="nav__link"
+            >Samsung</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Смарфоны и телефоны</a>
+            <a
+              href="#"
+              class="nav__link"
+            >Смарфоны и телефоны</a>
           </li>
           <li class="nav__item">
-            <a href="" class="nav__link">Ноутбуки и компьютеры</a>
+            <a
+              href="#"
+              class="nav__link"
+            >Ноутбуки и компьютеры</a>
```

Comment:
```suggestion
            <a
              href="#"
              class="nav__link"
            >
              Ноутбуки и компьютеры
            </a>
```
or like that for long one - fix it for other links